python/doc/examples/reliability_sensitivity/central_tendency.ipynb

```diff
@@ -0,0 +1,164 @@
+      "text/plain": [
+       "class=Graph name=Unnamed implementation=class=GraphImplementation name=Unnamed title=Importance Factors from Taylor expansions - Y xTitle= yTitle= axes=OFF grid=OFF legendposition= legendFontSize=1 drawables=[class=Drawable name=Unnamed implementation=class=Pie name=Unnamed labels=[E : 0.0%,F : 88.5%,L : 8.0%,I : 3.5%] radius=1 center=class=Point name=Unnamed dimension=2 values=[0,0] color palette=[#ff0000,#ccff00,#00ff66,#0066ff] derived from class=DrawableImplementation name=Unnamed legend= data=class=Sample name=Unnamed implementation=class=SampleImplementation name=Unnamed size=4 dimension=1 data=[[0.000353857],[0.884642],[0.079618],[0.0353857]] color=blue fillStyle=solid lineStyle=solid pointStyle=plus lineWidth=1]"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Perform taylor approximation to get mean of Y, importance factors\n",
+    "taylor = ot.TaylorExpansionMoments(Y)\n",
+    "taylor_mean_fo = taylor.getMeanFirstOrder()\n",
+    "taylor_mean_so = taylor.getMeanSecondOrder()\n",
+    "taylor_cov = taylor.getCovariance()\n",
+    "taylor_if = taylor.getImportanceFactors()\n",
+    "print('taylor mean first order=', taylor_mean_fo)\n",
+    "print('taylor importance factors=', taylor_if)\n",
+    "taylor.drawImportanceFactors()"
```

Comment:
Variance also?
It might be important to compare number of functionalists/grad/hessian calls and compare it with Monte-Carlo